src/library-authoring/component-info/ComponentManagement.test.tsx

```diff
@@ -76,15 +80,25 @@ describe('<ComponentManagement />', () => {
     render(mockLibraryBlockMetadata.usageKeyNeverPublished);
     expect(await screen.findByText('Draft')).toBeInTheDocument();
     expect(await screen.findByText('(Never Published)')).toBeInTheDocument();
-    expect(screen.getByText(matchInnerText('SPAN', 'Draft saved on June 20, 2024 at 13:54.'))).toBeInTheDocument();
+    expect(
+      screen.getByText(
+        matchInnerText('SPAN', 'Draft saved on June 20, 2024 at 13:54.'),
+      ),
+    ).toBeInTheDocument();
   });
 
   it('should render published status', async () => {
     render(mockLibraryBlockMetadata.usageKeyPublished);
     expect(await screen.findByText('Published')).toBeInTheDocument();
     expect(screen.getByText('Published')).toBeInTheDocument();
     expect(
-      screen.getByText(matchInnerText('SPAN', 'Last published on June 22, 2024 at 24:00 by Luke.')),
+      screen.getByText(
+        (_: string, element: Element | null) => !!element
+                && element.nodeName === 'SPAN'
+                && /Last published on June 22, 2024 at (00:00|24:00) by Luke\./.test(
+                  getInnerText(element),
+                ),
+      ),
```

Comment:
```suggestion
      screen.getByText(matchInnerText('SPAN', 'Last published on June 22, 2024 at 00:00 by Luke.')),
```
Could we revert this to the simpler (and more correct) check, once we've dropped Node 20 ?